tradingdb/relationaldb/models.py

```diff
@@ -2,12 +2,15 @@
 from django.db import models
 from model_utils.models import TimeStampedModel
 
-# ==================================
-#       Abstract classes
-# ==================================
 
 # Ethereum addresses have 40 chars (without 0x)
-ADDRESS_LENGTH = 40
+ADDRESS_LENGTH: str = 40
```

Comment:
This is an `int`, not `str`

Reply:
ohh, yes you're right, my fault! Thanks @Uxio0